src/citrine/resources/data_concepts.py

```diff
@@ -702,28 +728,32 @@ def list_by_tag(self, tag: str, per_page: int = 100) -> Iterator[ResourceType]:
             params=params)
         return (self.build(raw) for raw in raw_objects)
 
-    def delete(self, uid: Union[UUID, str], scope: str = CITRINE_SCOPE, dry_run: bool = False):
+    def delete(self, uid: Union[UUID, str, LinkByUID, BaseEntity],
+               scope: Optional[str] = None, dry_run: bool = False):
         """
-        Delete the element of the collection with ID equal to uid.
+        Delete an element of the collection by its id.
 
         Parameters
         ----------
-        uid: Union[UUID, str]
-            The ID.
-        scope: str
+        uid: Union[UUID, str, LinkByUID, BaseEntity]
+            A representation of the object (Citrine id, LinkByUID, or the object itself)
+        scope: Optional[str]
+            [DEPRECATED] use a LinkByUID to specify a custom scope
             The scope of the uid, defaults to Citrine scope (CITRINE_SCOPE)
         dry_run: bool
             Whether to actually delete the item or run a dry run of the delete operation.
             Dry run is intended to be used for validation. Default: false
 
         """
-        path = self._get_path() + "/{}/{}".format(scope, uid)
+        link = _make_link_by_uid(uid)
+        path = self._get_path() + "/{}/{}".format(link.scope, link.id)
         params = {'dry_run': dry_run}
         self.session.delete_resource(path, params=params)
         return Response(status_code=200)  # delete succeeded
 
-    def _get_relation(self, relation: str, uid: Union[UUID, str], scope: str,
-                      forward: bool = True, per_page: int = 100) -> Iterator[ResourceType]:
+    def _get_relation(self, relation: str, uid: Union[UUID, str, LinkByUID, BaseEntity],
+                      scope: Optional[str] = None, forward: bool = True, per_page: int = 100
+                      ) -> Iterator[ResourceType]:
```

Comment:
Heck of a signature.  Implement the breaking change of invalidating positional arguments?

Reply:
It's coming.

Fortunately this is an internal method, so the damage is contained